### PR TITLE
[MEDIUM-2897] android: updating lib and simplifying sample and adding medium theme

### PR DESCRIPTION
**Pull request by grosalex**
Related to [MEDIUM-2897](https://medium.atlassian.net/browse/MEDIUM-2897).

<details>
<summary>REVIEWERS (4)</summary>

R=@asaudemont
R=@christophespy007
R=@MickaelGuilbeaud
R=@prcaen
</details>

Manual testing: 🚫

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -10,14 +10,22 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+  <!--      <activity android:name=".MainActivity"
```
Comment (grosalex) [inline app/src/main/AndroidManifest.xml:13]: simplifying the sample

File: app/src/main/java/com/pddstudio/highlightjs/demo/SyntaxActivity.java [hunk 2]
```diff
@@ -33,16 +34,16 @@ public class SyntaxActivity extends AppCompatActivity implements
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_syntax);
-        FileObject fileObject = (FileObject) getIntent().getExtras().getSerializable("fileObject");
-        if(getActionBar() != null) {
-            assert fileObject != null;
-            getActionBar().setTitle(fileObject.getFileName());
+//        FileObject fileObject = (FileObject) getIntent().getExtras().getSerializable("fileObject");
```
Comment (grosalex) [inline app/src/main/java/com/pddstudio/highlightjs/demo/SyntaxActivity.java:37]: simplifying the sample

File: app/src/main/res/layout/activity_syntax.xml [hunk 3]
```diff
@@ -13,10 +12,22 @@
...
         android:layout_height="match_parent">
 
-        <com.pddstudio.highlightjs.HighlightJsView
-            android:id="@+id/highlight_view"
+        <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
-            android:layout_height="match_parent" />
+            android:layout_height="match_parent">
+
+            <com.pddstudio.highlightjs.HighlightJsView
```
Comment (grosalex) [inline app/src/main/res/layout/activity_syntax.xml:19]: making the sample look like a little bit more what we need

File: app/src/main/java/com/pddstudio/highlightjs/demo/SyntaxActivity.java (hunk 2 above)
Comment (MickaelGuilbeaud) [reply app/src/main/java/com/pddstudio/highlightjs/demo/SyntaxActivity.java:37]: IMO you can remove commented code if you don't think we will need it in the future.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (MickaelGuilbeaud) [inline app/src/main/AndroidManifest.xml:13]: IMO you can remove commented code if you don't think we will need it in the future.

**Review by MickaelGuilbeaud — APPROVED** [review]
LGTM

**Pull request merged**